src/app/common-ui/ErrorContent/index.js

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+// Components
+import ErrorBox from 'src/app/common-ui/ErrorBox';
+
+// Utils
+import styled from 'styled-components';
+
+const Container = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 50px 0;
+`;
+
+function ErrorContent({ content }) {
+  return (
+    <Container>
+      <ErrorBox content={content} />
+    </Container>
+  );
+}
+
+ErrorContent.propTypes = {
+  content: PropTypes.string,
+};
+
+export default ErrorContent;
```

Comment:
can put this file under `src/app/container/ProfilePage/components/ErrorContent/index.js`